New package: ITKIOWrapper v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: ITKIOWrapper
- Repository: https://github.com/JuliaHealth/ITKIOWrapper.jl
- Created by: @jakubMitura14
- Version: v0.0.1
- Commit: f3a07f6cc05cae2c7d04f880ed63f1fae88eec98
- Reviewed by: @jakubMitura14
- Reference: https://github.com/JuliaHealth/ITKIOWrapper.jl/commit/f3a07f6cc05cae2c7d04f880ed63f1fae88eec98#commitcomment-146392494
- Description: ITKIOWrapper.jl provides wrapper functions in Julia from the InsightToolkit Image Registration functions.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6525435d9c3ffaa5f216ef4606c4643b669e65699270de9d567854515fe1a6c13c1ec65576e517183df4ad2fee5e61d6766fea22c9a0508bc42da732d331210c526e79e8030c29be859c88d9552da6db7e6751a74c5e8a9c68c6b265c30c6254c2a27c30cce8e542342b33604d9fb81685ee4995e3fd8f7fc93d8972634ac8fde417e888dd7e00901a4325bccc7dbde629ea0b15946ef365db7b4c288abf0fa0f5595bb866efd2ec90cb42a84aec8563d -->

jakubMitura14 commented:
referenced from https://github.com/JuliaRegistries/General/pull/114712 - as suggested also in slack not getting new packages but working on original one

JuliaRegistrator committed:
New package: ITKIOWrapper v0.0.1

UUID: 8538b10a-c1b7-4e8d-b607-b3a5053e6a1f
Repo: https://github.com/JuliaHealth/ITKIOWrapper.jl.git
Tree: 4a758d53e0174307be9d69ce6dccba8ca2861041

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

jakubMitura14 commented:
Hello @goerz this is the same package you had analyzed earlier we changed project toml including version

goerz commented:
[noblock] Yes, I saw. You have to edit your comment to contain [noblock]
